### DISCUSSION TITLE
How to deliver a fix for an issue

### DISCUSSION THREAD

**Discussion by tonyyng** [EdgeTX/edgetx#2172] [Q&A] [answered]
Following the instructions for using gitpod for building the firmware, I tested a fix for this issue: https://github.com/EdgeTX/edgetx/issues/1991

It is a 3 line fix to logs.cpp at line 328 (after the block for UNIT_DATETIME):
```
            else if (sensor.unit == UNIT_TEXT) {
              f_printf(&g_oLogFile, "\"%s\",", telemetryItem.text);
            }
```
My repo is based on https://github.com/EdgeTX/edgetx/tree/v2.7.1 as opposed to main.

Are there instructions for getting this change from gitpod into the main repo and generating a pull request?  I'd be grateful if someone could provide the detailed steps or just deliver the change with work they are doing.

**Comment by gagarinlg** [comment, marked as answer]
1. You create your own fork of EdgeTX
2. You create a branch, based on main
3. You commit to you branch
4. You go to the EdgeTX repo Voe the github web page
5. Create PR

**Comment by tonyyng** [comment]
Thanks.  Done.
